clang/test/Analysis/element-region-address-space.c

```diff
@@ -1,11 +1,26 @@
 // RUN: %clang_analyze_cc1 -triple amdgcn-unknown-unknown \
-// RUN:   -analyzer-checker=core -verify %s
+// RUN:   -analyzer-checker=core,unix -verify %s
```

Comment:
```suggestion
// RUN:   -Wno-incompatible-library-redeclaration \
// RUN:   -analyzer-checker=core,unix -verify %s
```
You need to disable this compiler warning because otherwise the test fails (at least on my machine).

Reply:
Thanks for pointing this out, applied!